### PR TITLE
Learning git 01

### PR DESCRIPTION
**Pull request by MTTParry**
Stephen and I are playing with merging in the JS file

### CODE REVIEW

**Comment by sfurlani** [comment]
Looks very similar to #1 (Example Comment)

File: example.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const zip = (a, b) => a.map((k, i) => [k, b[i]]);
+
+// A change from the merge01 branch
+
+// A function to Add two numbers together
+function add(x, y) {
+    return x + y
+}
+
+// Adding a comment
```
Comment (sfurlani) [inline example.js]: ❗ Please Remove unneeded comment

File: example.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+const xs = [1, 2, 3, 4, 5, 6, 7]
+const ys = [1, 2, 3, 4, 5, 6, 7]
+var zs = []
+for (let i = 0; i < xs.count; i++) {
+    zs.push(add(xs[i], ys[i]))
+}
+console.log(zs)
+
+// Another Change in example01
```
Comment (sfurlani) [inline example.js:20]: ❓ Please explain why this is necessary.

**Review by sfurlani — CHANGES_REQUESTED** [review]

File: example.js (hunk 2 above)
Comment (MTTParry) [reply example.js:20]: This relates to requirement 10.a

Comment (sfurlani) [reply example.js:20]: Ok, thanks for clarifying

**Review by sfurlani — APPROVED** [review]

**Pull request merged**